new_aboutme/about_me2/header.php

```diff
@@ -0,0 +1,56 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+	<!-- Required meta tags -->
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+
+	<!-- Bootstrap CSS -->
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+	<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat">
+	<link rel="stylesheet" href="css/aboutme2.css">
+
+	<title></title>
+</head>
```

Comment:
None of this is needed when just including a snippet. When you do the `inlcude()` function it is essentially dropping the  contents of that file right in to the other one. So if you think about where it is included, you now have an entire HTML doc inside of the original one.